packages/app/src/debug/DebugPageHeader.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <div
+    data-cy="debug"
+    class="flex flex-col pb-24px gap-16px"
+  >
+    <div
+      :data-cy="`debug-header-${debug.id}`"
+      class="grid px-24px w-full overflow-hidden flex items-center border gap-y-2 py-24px"
+    >
+      <ul
+        :data-cy="'header-top'"
+        class="flex self-stretch items-center gap-x-2 flex-row whitespace-nowrap"
+      >
+        <li
+          v-if="debug?.commitInfo?.summary"
+          class="font-medium text-lg text-gray-900"
+          data-cy="debug-test-summary"
+        >
+          {{ debug.commitInfo.summary }}
+        </li>
+        <div
+          v-if="defaultscommitInfo.runNumber && commitsAhead"
+          class="border rounded border-gray-100 items-center text-sm h-6"
+          data-cy="debug-runCommit-info"
+        >
+          <span
+            class="font-medium text-gray-700 border-r-1px px-2 mx-px items-center"
+            data-cy="debug-runNumber"
+          >
+            {{ defaultscommitInfo.runNumber }}
+          </span>
+          <span
+            class="font-normal text-orange-500 px-2 mx-px items-center"
+            data-cy="debug-commitsAhead"
+          >
+            {{ commitsAhead }}
+          </span>
+        </div>
+        <li class="-mt-8px text-lg text-gray-400">
+          .
+        </li>
+        <li class="font-normal text-sm text-indigo-500">
+          <ExternalLink
+            :data-cy="'debug-header-dashboard-link'"
+            :href="debug.url || '#'"
+            :use-default-hocus="false"
+          >
+            <span class="sr-only">Dashboard Link:</span> View in the dashboard
+          </ExternalLink>
+        </li>
+      </ul>
+      <ul
```

Comment:
This dot separated list is the same in the runs page. Can we pull this into a reusable component?

Reply:
After more discussion with @warrensplayer  I don't think we want a reusable component here. Despite the similarity between the runs page and this component, there are design parts that are completely distinct. For example: the number of components that show on the separated list and the order in which they show up. Furthermore,  on the runs page, the dot separated list is part of the `listRowHeader` component which also accepts a "Header" section which is not a part of this component. So leaving this as a standalone component would suffice